draft-ietf-quic-http.md

```diff
@@ -659,6 +704,15 @@ CANCEL_PUSH frame, the client can abort reading the stream with an error code of
 H3_REQUEST_CANCELLED. This asks the server not to transfer additional data and
 indicates that it will be discarded upon receipt.
 
+Pushed responses that are cacheable (see Section 3 of {{!RFC7234}}) can be
```

Comment:
This is okay for now, but it should be improved eventually to talk about browser state separately from the cache. IOW, define push as part of the browser history (prehistory) because this is entirely separate from a cache (even if they use the same storage handles).

Reply:
Push isn't part of the cache, but it does use the definition of "cacheable" -- do you think it should be pointing elsewhere?

Reply:
This is a _very_ murky area, still.

Reply:
Separating to #3530 so I can merge this.